src/tasks/tasksHelper.ts

```diff
@@ -0,0 +1,72 @@
+import * as vscode from 'vscode';
+import { BuildTaskProvider } from './buildTaskProvider';
+import { TaskPseudoTerminal } from './taskPseudoTerminal';
+import { androidHelper, iOSHelper, Helpers } from './helpers';
+import { PackageTaskProvider } from './packageTaskProvider';
+import { quickPick } from '../quickpicks';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/tasks/tasksHe...
  
  
  DangerID: danger-id-Danger;
  File: src/tasks/tasksHelper.ts;
  Line: 6;
-->

- :warning: src/tasks/tasksHelper.ts line 6 – 'quickPick' is defined but never used. (@typescript-eslint/no-unused-vars)